models/articles.model.js

```diff
@@ -49,3 +43,15 @@ ORDER BY
             return data.rows;
         });
 };
+exports.selectComments = article_id => {
+    if (isNaN(article_id)) {
```

Comment:
we should already get an error from psql if this is the case, then our psql error handler should take care of this